cabal-install/src/Distribution/Client/ProjectPlanning.hs

```diff
@@ -460,6 +455,47 @@ rebuildProjectConfig
             projectConfigBuildOnly
             pkgLocations
 
+      informAboutConfigFiles projectConfig = do
+        cwd <- getCurrentDirectory
+        let out -- output mode is verbose ('notice') if we build outside the project root
+              | cwd == distProjectRootDirectory = info
+              | otherwise = notice
+        unless (null configFiles)
+          . out (verboseStderr verbosity)
+          . render
+          $ message
+        where
+          -- message formatting depends on |config files| (the number of config files)
+          message = case configFilesDoc of
+            (_ : _ : _ : _) ->
+              -- \|config files| > 2 => vertical list
```

Comment:
I've seen this before and it is a fourmolu artifact, isn't it, the `-- \|`.

Reply:
I think fourmolu only allows `-- |` at the top level, as a haddock comment.

Reply:
Ah, quite possible, thank you for the hint!